src/Microsoft.DotNet.Tools.Publish/Program.cs

```diff
@@ -31,6 +31,7 @@ public static int Main(string[] args)
             var output = app.Option("-o|--output <OUTPUT_PATH>", "Path in which to publish the app", CommandOptionType.SingleValue);
             var configuration = app.Option("-c|--configuration <CONFIGURATION>", "Configuration under which to build", CommandOptionType.SingleValue);
             var project = app.Argument("<PROJECT>", "The project to publish, defaults to the current directory. Can be a path to a project.json or a project directory");
+            var nativeSubdirectories = app.Option("--native-subdirectory", "Include subdirectories from native assets of dependency packages in output", CommandOptionType.NoValue);
```

Comment:
This definitely seems a little hacky, can we get a tracking bug to clean it up? I don't mind hacks as long as we track removing them ;P